.gitignore

```diff
@@ -0,0 +1,11 @@
+/.bundle/
+/.yardoc
+/_yardoc/
+/coverage/
+/doc/
```

Comment:
ya know, I once got a PR rejected for that because "That should be in your global gitignore anyways, not in this project". I happen to disagree with that mentality so I'm adding it.